src/test/scala/mesosphere/marathon/core/instance/TestInstanceBuilder.scala

```diff
@@ -170,6 +174,27 @@ object TestInstanceBuilder {
     None
   )
 
+  def runningInstance(appId: PathId, version: Timestamp, instanceTracker: InstanceTracker): Future[(Instance.Id, Task.Id)] = async {
```

Comment:
Could you add some docs?

I'm not sure if we really should introduce this helper. As I understand we want to save a running instance in the tracker, right? Ideally this should be a two liner. Could we create a running instance with the test builder?